src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.tsx

```diff
@@ -240,7 +270,7 @@ const TranscriptSummary = () => {
         </div>
       </div>
 
-      {transcript.so_term === 'protein_coding' && (
+      {metadata.biotype?.value === 'protein_coding' && (
```

Comment:
why the question mark?

Reply:
good catch. Biotype should always be there